blog/en/blog/2023/10/17/release-apache-apisix-3.6.1.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Release Apache APISIX 3.6.1"
+authors:
+  - name: "Ashish Tiwari"
+    title: "Author"
+    url: "https://github.com/Revolyssup"
+    image_url: "https://avatars.githubusercontent.com/u/43276904?v=4"
+  - name: "Traky Deng"
+    title: "Technical Writer"
+    url: "https://github.com/kayx23"
+    image_url: "https://avatars.githubusercontent.com/u/39619599?v=4"
+keywords:
+- Apache APISIX
+- API Gateway
+- API Management Platform
+- New Release
+- Cloud Native
+description: The Apache APISIX 3.6.1 version is released on October 17, 2023. This release includes a few bug fixes.
+tags: [Community]
+---
+
+We are glad to present Apache APISIX 3.6.1 with a number of bug fixes and continuous improvements to user experiences.
```

Comment:
```suggestion
> We are glad to present Apache APISIX 3.6.1 with a number of bug fixes and continuous improvements to user experiences.
```

Reply:
Thanks for the review. But why?

Reply:
- To tell apart the summary and content on the blog page, i.e. 

![image](https://github.com/apache/apisix-website/assets/36651058/6bb780bd-fbcf-41dc-8d78-ae0e0fc1399b)

- To remind writers don't forget to add `<!--truncate-->` after summary.